File: vSongApp/src/main/java/com/jackson_siro/visongbook/models/SelectableBook.java
package com.jackson_siro.visongbook.models;

public class SelectableBook extends CategoryModel {
    private boolean isSelected = false;

    public SelectableBook(CategoryModel slider, boolean isSelected) {
        super(slider.getBookid(), slider.getCategoryid(), slider.getTitle(), slider.getTags(), slider.getQcount(),
                slider.getPosition(), slider.getContent(), slider.getBackpath());
        this.isSelected = isSelected;
    }

    public boolean isSelected() {
        return isSelected;
    }

    public void setSelected(boolean selected) {
        isSelected = selected;
    }
}

File: vSongApp/src/main/java/com/jackson_siro/visongbook/models/SelectableBook.kt
package com.jackson_siro.visongbook.models

class SelectableBook(slider: CategoryModel, isSelected: Boolean) : CategoryModel(slider.bookid, slider.categoryid, slider.title, slider.tags, slider.qcount, slider.position, slider.content, slider.backpath) {
    var isSelected = false

    init {
        this.isSelected = isSelected
    }
}